pandas/core/interchange/column.py

```diff
@@ -194,6 +211,13 @@ def describe_null(self):
             column_null_dtype = ColumnNullType.USE_BYTEMASK
             null_value = 1
             return column_null_dtype, null_value
+        if isinstance(self._col.dtype, ArrowDtype):
+            if all(
```

Comment:
Is there a real need to iterate the chunks like this and check null / not-null? Arrow leaves it implementation defined as to whether or not there is a bitmask

https://arrow.apache.org/docs/format/Columnar.html#validity-bitmaps

I just wonder if there is value in use trying to dictate that through the interchange protocol versus letting consumers handle that

Reply:
I've changed things round a bit to just rechunk upfront (if `allow_copy` allows)

so once get here, we just need to check if `buffers()[0]` is `None`

The issue with just returning `ColumnNullType.USE_BITMASK, 0` in all cases, even if there's no validity mask, is that then `pyarrow.interchange.from_dataframe` would raise